benchmarks/python/test_huggingface_attn_fwd.py

```diff
@@ -71,9 +71,26 @@ def huggingface_attn_fwd_fusion(
     fd.add_output(T47)
 
 
+def huggingface_attn_fwd_fn(
```

Comment:
Might I suggest
```suggestion
def huggingface_attn_fwd(
```
And related changes further down in this file.